### ISSUE TITLE
Add User Download Limit

### ISSUE THREAD

**Issue by emohandesi** [aleskxyz/reality-ezpz#96]
Please add an option to set a limit on how much data can a user download.

**Comment by emohandesi** [comment]
@cpalll! While Alireza is working on the implementation of this feature, you can use the [Sanai Panel](https://github.com/MHSanaei/3x-ui) that supports the Reality protocol in addition to putting a limit on the download of each user.

**Comment by cpalll** [comment]
> @cpalll! While Alireza is working on the implementation of this feature, you can use the [Sanai Panel](https://github.com/MHSanaei/3x-ui) that supports the Reality protocol in addition to putting a limit on the download of each user.

Does this involve setting up xray again using the Panel or is it compatible with reality-ezpz.

**Comment by emohandesi** [comment]
> > @cpalll! While Alireza is working on the implementation of this feature, you can use the [Sanai Panel](https://github.com/MHSanaei/3x-ui) that supports the Reality protocol in addition to putting a limit on the download of each user.
> 
> Does this involve setting up xray again using the Panel or is it compatible with reality-ezpz.

I do not think it is compatible with reality-ezpz. I have not set up XRay before. I simply install the Sanai panel and it gives me an administration link. Through that, I can create users in bulk with data caps for them.
